packages/templates/app/rust/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "templates-app-rust",
+  "description": "Polywrap App Rust Template",
+  "private": true,
+  "version": "0.11.2",
+  "scripts": {
+    "build": "npx polywrap codegen",
+    "test": "ts-node ./src/index.ts",
+    "ipfs:up": "npx polywrap infra up --modules=eth-ens-ipfs",
```

Comment:
this isn't needed

Reply:
Forget to remove! before I was using local ipfs in template but then migrate to just cat instead of add and cat